data/ecosystem/adopters.yaml

```diff
@@ -4,6 +4,11 @@
   components: [Collector]
   reference: ''
   contact: ''
+- name: Bureau.id
+  url: https://bureau.id/
+  components: [Collector, Python, Golang, Lambda]
+  reference: 'https://tech.bureau.id/connecting-the-dots-with-opentelemetry-part-ii-5ea5f6b06c29'
+  contact: '' 
```

Comment:
Can you put a contact here, this is for us (maintainers) to know whom to reach if for example that link expires, can be your github handle (preferred) or an email address